src/ui/main.cpp

```diff
@@ -60,10 +70,12 @@ int main(int argc, char *argv[])
     } else if (translator.load(QLocale(langCode),
                                QString("%1").arg(qApp->applicationName().toLower()),
                                QString("_"),
-                               QString("%1/../../share/%2/translations")
+                               QString("%1/../share/%2/translations")
```

Comment:
You're right... this actually breaks the installation on my end (when installed)...  But according to #281 its already broken for another user, though I'm sure there's special circumstances involved.

I'll go ahead and revert this small change so it'll just be a feature request rather than a bug fix, but we need to see about a more reliable way to detect this once installed on the system in the future.